File: week2/day2/week2-day2/week2-day2/Counter.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace week2_day2
{
    class Counter
    {
        public int count;
        public int initial;
        public Counter(int count)
        {
            this.count = count;
            this.initial = count;
        }

        public Counter():this(0)
        {
            
        }

        public void Add(int number)
        {
            this.count += number;
        }

        public void Add()
        {
            this.count++;
        }

        public string Get()
        {
            return count.ToString();
        }

        public void Reset()
        {
            this.count = this.initial;
        }
    }
}

File: DSA-2019/week-02/day-1/counter.py
class Counter:
    def __init__(self, num=0):
        self.number = int(num)
        self.initial_num=num
    def add(self, num=1):
        self.number += num

    def get(self):
        return self.number

    def reset(self):
        self.number=self.initial_num
